packer_templates/sles/sles-15-sp1.json

```diff
@@ -25,19 +25,9 @@
       "ssh_username": "vagrant",
       "ssh_wait_timeout": "10000s",
       "type": "virtualbox-iso",
-      "vboxmanage": [
-        [
-          "modifyvm",
-          "{{.Name}}",
-          "--memory",
-          "{{ user `memory` }}"
-        ],
-        [
-          "modifyvm",
-          "{{.Name}}",
-          "--cpus",
-          "{{ user `cpus` }}"
-        ],
+      "memory": "{{ user `memory` }}",
+      "cpus": "{{ user `cpus` }}",
+      "manager": [
```

Comment:
This line is causing packer validate failures

Reply:
Ah crap. That's my bad, this is one of the ones I manually renamed and intended to rename back. Thanks for catching this. Gimme one sec for one more commit before you continue to review this.